src/events/handlers/CountingHandler.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Events,
+  type ClientEvents,
+  Message,
+  type OmitPartialGroupDMChannel,
+  type EmojiIdentifierResolvable, Emoji, GuildEmoji, ReactionEmoji, type User, type Interaction
+} from "discord.js";
+import {Logger, type ILogObj} from "tslog";
+import type IEvent from "../IEvent.ts";
+import type SnugglesClient from "../../structure/Client.ts";
+import GuildCountingService from "../../database/services/GuildCountingService.ts";
+import {column, count, evaluate, number, re} from 'mathjs'
+import type {GuildCounting} from "@prisma/client";
+
+
+export const logger: Logger<ILogObj> = new Logger();
+
+export class CountingHandler implements IEvent {
+  public event: keyof ClientEvents = Events.MessageCreate;
+  public once = false;
+
+  constructor(private client: SnugglesClient) {}
+
+  async handle(message: OmitPartialGroupDMChannel<Message<boolean>>) {
+    const countData = await GuildCountingService.getCountingDataForChannel(message.channelId);
+    if (!countData) return;
+    logger.debug(`message: ${message.content} | count: ${countData.count}`)
```

Comment:
we can't have debug code in production...

Reply:
mY BAD